client/src/modules/patients/edit/edit.html

```diff
@@ -108,6 +108,18 @@
                   </div>
                 </div>
 
+                 <!-- Reference Name -->
+                 <div
+                 class="form-group has-feedback"
+                 ng-class="{'has-error' : patientDetails.display_name.$invalid && patientDetails.$submitted}">
```

Comment:
This is better!

Since the user can't edit it, we don't need the `has-feedback` or `has-error` classes.  go ahead and remove those.

Reply:
This still needs to be addressed, and then we can merge it.

Reply:
@jeremielodi are you going to address these?